index.html

```diff
@@ -2,10 +2,21 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Old+Standard+TT|VT323" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
```

Comment:
Good work setting up your CSS, but try to break it out into files that conform with SMACSS principles for this lab, even if you don't end up doing much styling yet.